python/data_structures/linked_list.py

```diff
@@ -1,6 +1,7 @@
 class LinkedList:
     """
     New Implementation for linked list
+    Create a Node class that has properties for the value stored in the Node, and a pointer to the next Node.
```

Comment:
The node docstring belongs with the node class. at line 38. 
`
class Node:
    """
    Properties for the value stored in the Node, and a pointer to the next Node.
    """
    def __init__(self, value, next=None):
        self.value = value
        self.next = next
`